addons/hr_holidays/static/src/owl/components/thread/thread.js

```diff
@@ -0,0 +1,51 @@
+odoo.define('hr_holidays.component.Thread', function (require) {
+'use strict';
+
+var Thread = require('mail.component.Thread');
```

Comment:
no `var`, use `const`, and relax to `let` only when necessary

Reply:
changed, mistake from copy/paste/adapation from old stuff